apollo-compiler/src/main/kotlin/com/apollostack/compiler/ir/Field.kt

```diff
@@ -0,0 +1,20 @@
+package com.apollostack.compiler.ir
+
+class Field(val responseName: String, val fieldName: String, val type: String,
```

Comment:
Should we make this class as data class?
And not sure if the `equals` method implemented correctly, as it takes into account only `type` but there can be the case when 2 fields have the same `type` but different let say `fieldName`, so are they equal?